src/Microsoft.ML.Transforms/Text/LdaTransform.cs

```diff
@@ -0,0 +1,978 @@
+﻿//------------------------------------------------------------------------------
+// <copyright company="Microsoft Corporation">
+//     Copyright (c) Microsoft Corporation. All rights reserved.
+// </copyright>
+//------------------------------------------------------------------------------
+
+using Float = System.Single;
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.CommandLine;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.EntryPoints;
+using Microsoft.ML.Runtime.Internal.Internallearn;
+using Microsoft.ML.Runtime.Internal.Utilities;
+using Microsoft.ML.Runtime.Model;
+using Microsoft.ML.Runtime.TextAnalytics;
+
+[assembly: LoadableClass(typeof(LdaTransform), typeof(LdaTransform.Arguments), typeof(SignatureDataTransform),
+    LdaTransform.UserName, LdaTransform.LoaderSignature, LdaTransform.ShortName, DocName = "transform/LdaTransform.md")]
+
+[assembly: LoadableClass(typeof(LdaTransform), null, typeof(SignatureLoadDataTransform),
+    LdaTransform.UserName, LdaTransform.LoaderSignature)]
+
+/*
+ * REVIEW taifengw(tfinley): So this would be my starting point. Much of the plumbing is already
```

Comment:
>taifengw(tfinley) [](start = 10, length = 17)

strip all aliases from code

Reply:
d

---
In reply to: [196948610](https://github.com/dotnet/machinelearning/pull/377#discussion_r196948610) [](ancestors = 196948610)